setup.py

```diff
@@ -52,6 +52,7 @@ def read(*rnames):
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
```

Comment:
I think this is a bit too early. Did you look into the changes in Python 3.7 and the consequences for RestrictedPython? Successfully running tests do not mean that using this version is as secure as the other versions.